### ISSUE TITLE
[FEATURE] Implement waiting for`ctl scale actor`

### ISSUE THREAD

**Issue by ghost** [wasmCloud/wash-archive#251]
Taken from PR #245:

One thing I'm considering would be useful here, especially for the actor start/stop events, is a `count` parameter where you can wait for a specified number of events. 

For example, if you do a `wash ctl start actor <echo> --count 5`, the ideal scenario would be to wait for 5 `ActorStarted` events. 

I'm personally willing to file that as a feature request though and it doesn't need to be done in this PR. I think I lean towards the feature request only because waiting for multiple events introduces a few questions that I don't have a good answer to, e.g.
1. What if you receive 3/5 successful events before the timeout, is that a mixed success or a fail?
2. What if you receive 99 successful events and 1 failure?
3. What if you receive 1 successful event and 99 failures?

TL;DR I think we feature request awaiting multiple as there are some nuances

_Originally posted by @brooksmtownsend in https://github.com/wasmCloud/wash/pull/245#discussion_r830016325_
